arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/Hub.java

```diff
@@ -669,6 +672,8 @@ public void traceContextExit(MessageFrame frame) {
         this.callStack.revert(this.state.stamps().hub()); // TODO: Duplicate s?
       }
 
+      defers.resolveUponExitingContext(this, currentFrame());
+
```

Comment:
We were somehow missing this ?

Reply:
no, it's already done at the end of `traceContextExit` 
![image](https://github.com/user-attachments/assets/8dbb8ebe-50ad-466a-bac7-b90596d0af72)

Reply:
we need to do it for any depth, I delete your add.

Reply:
https://github.com/Consensys/linea-tracer/pull/1027/commits/b859713fabb537eefc15cb75e16ade62fa630f0b

Reply:
Indeed